RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -258,4 +258,71 @@ private extension CustomerCenterConfigData.Screen {
 
 }
 
+private extension CustomerCenterConfigData.HelpPath {
+    func isWithinRefundWindow(purchaseInformation: PurchaseInformation?, now: Date) -> Bool {
+        guard let purchaseInformation,
+              type != .refundRequest else {
+            return true
+        }
+
+        if let refundWindow {
+            switch refundWindow {
+            case .forever:
+                return true
+            case let .duration(duration):
+
+                return duration.withinDuration(referenceDate: purchaseInformation.latestPurchaseDate, now: now)
+            }
+        }
+
+        return true
+    }
+}
+
+private extension Array<CustomerCenterConfigData.HelpPath> {
+    func relevantPathsForPurchase(
+        _ purchaseInformation: PurchaseInformation?,
+        now: Date) -> [CustomerCenterConfigData.HelpPath] {
+
+        guard let purchaseInformation else {
+            return self
+        }
+
+        return filter { !purchaseInformation.isLifetime || $0.type != .cancel }
+                .filter { $0.refundWindow.map { $0.withinWindow(purchaseInformation, now: now) } ?? true }
+    }
+}
+
+private extension CustomerCenterConfigData.HelpPath.RefundWindowDuration {
+    func withinWindow(_ purchaseInformation: PurchaseInformation, now: Date) -> Bool {
+        switch self {
+        case .forever: return true
```

Comment:
```suggestion
        case .forever: 
            return true
```
nit: I think it's more consistent and readable